.github/workflows/build_wheels.yml

```diff
@@ -17,10 +20,13 @@ jobs:
         python3 -m build --sdist --wheel gen/python --outdir ./dist
 
     - name: Upload Wheels
-      uses: actions/upload-artifact@v3
+      if: startsWith(github.ref, 'refs/tags')
+      uses: pypa/gh-action-pypi-publish@master
       with:
-        name: symforce-wheels
-        path: dist/*
+        user: ${{ secrets.ARTIFACTORY_USER }}
+        password: ${{ secrets.ARTIFACTORY_PASSWORD }}
+        repository_url: https://zipline.jfrog.io/zipline/api/pypi/pypi
+        packages_dir: client/dist/
```

Comment:
I _think_ you'll want this to be just `dist/` since that's the output path for `python3 -m build...` above

Reply:
good catch! I just blindly copy-pasted haha. updated!